src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -877,7 +877,7 @@ public function dropDatabase($name)
      */
     protected function getDefaultValueDefinition($default)
     {
-        if (is_string($default) && 'CURRENT_TIMESTAMP' !== $default) {
+        if (is_string($default) && 'CURRENT_TIMESTAMP' !== $default && !preg_match('/^uuid_.*\(.*\)$/', $default)) {
```

Comment:
what about `&& !preg_match('/^.*\(.*\)$/', $default)`. This would then match any function. Although IMO it would be better to create "RawTokenInterface" or something that could be also reused elsewhere. If that would be provided, then the author of migration would be responsible for properly quoting things and phinx would just output what's provided